File: Arcade/Intro/Ruby/palindromeRearranging.rb
def palindromeRearranging(inputString)
    abc = Array.new(26, 0)
    inputString.size.times{|i| abc[inputString[i].ord - "a".ord] += 1}
    return inputString.size % 2 ==
    abc.map{|e| e % 2 == 1 ? 1 : 0}.sum
end

File: Arcade/Intro/Java/PalindromeRearranging.java
import java.util.Arrays;

final class PalindromeRearranging {

	boolean palindromeRearranging(String inputString) {
		int[] abc = new int[26];
	    for(int i = 0; i < inputString.length(); i++) {
	        abc[inputString.charAt(i) - 'a']++;
	    }	
	    
	    return inputString.length() % 2 ==
	    Arrays.stream(abc).map(n -> n % 2 == 1 ? 1 : 0).sum();
	}
}